microsoft-edge/extensions-chromium/store-policies/csp.md

```diff
@@ -305,9 +305,10 @@ While the initial script runs, the call to `eval` is blocked.  That is, while th
 Since content scripts aren't affected by the CSP of the page, this a great reason to put as much behavior as possible of your Extension into the content script, rather than DOM injected scripts.
 
 
+<!-- ====================================================================== -->
 > [!NOTE]
 > Portions of this page are modifications based on work created and [shared by Google](https://developers.google.com/terms/site-policies) and used according to terms described in the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0).
-> The original page is found [here](https://developer.chrome.com/extensions/contentSecurityPolicy).
+> The original page is found [here](https://developer.chrome.com/docs/extensions/reference/manifest/content-security-policy).
```

Comment:
6: [here] link - what's the best webpage to link to?  @Bhuvana-Priya

Reply:
Conclusion from MH & BP: the updated [here] link might be correct.  Multiple Chrome articles have headings and/or key words in common w/ the present article.

Maybe change the link from:
https://developer.chrome.com/docs/extensions/reference/manifest/content-security-policy
to:
https://developer.chrome.com/docs/privacy-security/csp

Search of Chrome docs for article title/key words "Content Security Policy":
https://developer.chrome.com/s/results?q=Content%20Security%20Policy#gsc.tab=0&gsc.q=Content%20Security%20Policy&gsc.sort=

Reply:
we kept the initial revised url and added hidden comment with hit 3 url